config/crg.scoreboard.properties

```diff
@@ -17,7 +17,6 @@
 
 # These are the servlets that are started.
 # Unless you know what you are doing, you should not modify this.
-com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.1 = com.carolinarollergirls.scoreboard.jetty.XmlScoreBoardServlet
 com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.2 = com.carolinarollergirls.scoreboard.jetty.ScoreBoardVersionServlet
 com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.servlet.5 = com.carolinarollergirls.scoreboard.jetty.MediaServlet
```

Comment:
I guess at this point we can just hardcode the last remaining values and get rid of the properties file.

Reply:
I've been planning to do that for some time.